### ISSUE TITLE
/ss checkedit used in DragonRealms

### ISSUE THREAD

**Issue by abirmingham** [pltrant/GSL-Editor#48]
This command is not supported in that instance, so the extension should avoid using it.

**Comment by abirmingham** [comment]
Hmmm. Yusri says that DR does now have `/ss checkedit`. Unfortunately I can't check as I don't have access - but we might want to check and revert this commit if true.

**Comment by pltrant** [comment]
This is not true. They only have it in their Dev instance and it requires being deployed to every instance (to check each instance's script status).
